### DISCUSSION TITLE
[预览]v2.0.4

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2575] [分享 / Share]
- 修复在线仓库无法加载的问题. (#2560)

由于 Tampermonkey beta 版本 (红猴) 大于等于 4.14.6147 时有 bug, 还是会报错, 请先换回非 beta 版 (黑猴). Violentmonkey 用户不受影响.

----

版本号更新策略调整: 虽然 v2 的版本号有 commit hash, 但不改前面的数字号确实不算真正意义上的更新 (油猴检查更新检查不到), 所以 v2 之后可能会有更高的发版频率 (相对的, 每次更新的内容量会减少).

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.0.4-preview'>[预览]v2.0.4</a>.</em>

**Comment by wangqs97** [comment]
adguard有测试过是否支持吗？我这里还是显示无法加载在线组件

**Reply by the1812** [reply]
没测过 (以前也只是凑巧支持
